tests/tests/txpool_api.rs

```diff
@@ -199,3 +199,83 @@ async fn test_txpool_status(#[future] katana: Katana, _setup: ()) {
     // Drop the server handle to shut down the server after the test
     drop(server_handle);
 }
+
+#[rstest]
+#[awt]
+#[tokio::test(flavor = "multi_thread")]
+async fn test_txpool_inspect(#[future] katana: Katana, _setup: ()) {
+    // Start the Kakarot RPC server and retrieve the server address and handle
+    let (server_addr, server_handle) =
+        start_kakarot_rpc_server(&katana).await.expect("Error setting up Kakarot RPC server");
+
+    // Generate a vector of random bytes
+    let bytes: Vec<u8> = (0..RANDOM_BYTES_SIZE).map(|_| rand::random()).collect();
+    let mut unstructured = arbitrary::Unstructured::new(&bytes);
+
+    // Generate 10 pending transactions and add them to the database
+    let mut pending_transactions = Vec::new();
```

Comment:
I think we have a lot of boiler plate in these tests no? Could we use a few helpers maybe?

Reply:
It's true but I told myself that:
- It was more flexible knowing that depending on the bugs that we encounter in the future, these testing conditions may evolve and not be the same for all unit tests.
- It was rather readable and compartmentalized like that.
- As we already have quite a few utils for testing, I didn't want to add too much.

Maybe we can do that on a follow up good first issue with a smart arrangement if necessary?

Reply:
Sure sounds good, I was just maybe thinking a simple function (doesn't even have to be in the test_utils) that returns pending transactions for example. But good first issue works!